act/virtualNetwork.tf

```diff
@@ -86,25 +86,55 @@ resource "azurerm_storage_account" "my_storage" {
   location                 = azurerm_resource_group.resource_group.location
   account_tier             = "Standard"
   account_replication_type = "LRS"
+
+  queue_properties {
+    logging {
+      write                 = true
+      read                  = true
+      version               = "1.0"
+      retention_policy_days = 365
+      delete                = true
+    }
+    hour_metrics {
+      enabled               = true
+      version               = "1.0"
+      include_apis          = true
+      retention_policy_days = 365
+    }
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure FTP state is set to disabled or FTPS only for Azure Function App**
 
**Vulnerability**
Azure FTP deployment endpoints are un-encrypted and open to public. An attacker listening to network traffic can see login traffic in clear text, granting them full control of the code base of the app or service.
 
**Remediation**
Azure Function App FTP state should be set to disabled or FTPS only. Deployment's FTP state can be modified to prevent MiTM attacks for Azure Function App.

source: accurics-beta